sympy/stats/tests/test_joint_rv.py

```diff
@@ -199,9 +191,24 @@ def test_NegativeMultinomial():
     raises(ValueError, lambda: NegativeMultinomial('b2', k0, 0.5, 0.4, 0.3, 0.4))
 
 
+def test_Wishart():
+    n, p = symbols('n p')
+    _k = Dummy('k')
+    A = MatrixSymbol('A', p, p)
+    V = MatrixSymbol('V', p, p)
+    X = Wishart('X', n, V)
+    density(X)(A).dummy_eq(2 ** (-n * p / 2) * pi ** (-p * (p - 1) / 4) * exp(-Trace(V ** (-1) * A) / 2) *
+                           Determinant(A) ** (n - p - 1) * (Determinant(V) ** (n / 2)) ** (-1) / Product(
+        gamma(-_k / 2 + n / 2 + 1 / 2), (_k, 1, p)))
+
+    X = Wishart('X', 2, [[1, 0], [0, 1]])
+    assert density(X)([[2, 0], [0, 2]]) == exp(-Trace(Matrix([[2, 0], [0, 2]]))/2)/(16*pi)
+    raises(ValueError, lambda: Wishart('X', 1, [[1, 0], [0, 1]]))
+    raises(ValueError, lambda: Wishart('X', 2, [[0, 1], [1, 0]]))
+
```

Comment:
Is it possible to add tests for marginal distributions of Wishart distribution?

Reply:
I will after completing `set` function

Reply:
Is there any way to perform integration over matrices via `sympy.integrate` or any other sympy API?